flutter/lib/desktop/widgets/remote_toolbar.dart

```diff
@@ -2003,6 +2009,7 @@ class _MultiMonitorMenu extends StatelessWidget {
         Obx(() {
           RxInt display = CurrentDisplayState.find(id);
           return _IconMenuButton(
+            tooltip: "Multi monitor",
```

Comment:
this tooltip can tell user nothing. Meaningless tooltip is worse than no tooltip.

Reply:
you should give a really helpful tooltip per context. if can not, we should remove it, not for so called consistency.